number_of_pods_in_system.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+# This script checks the number of pods in a given namespace
+# It waits untill we have the minimum number of pods to start working on the system
+# 
+# It was created simnce after nb install sometimes the backingstore is not created yet
+# In k8s we cannot wait for a resource that doesn't exist
+#
+# arguemnts (optional): namesspace, threshold pods number (minimum pods that we need to use the system)
+CURRENT_PODS_NUMBER=0
+THESHOLD_PODS_NUMBER=5
+NAMESPACE=default
+NUMBER_OF_ITERATIONS=15 # arbitrary number
+
+if [ $# -eq 0 ]; then
+  echo "Using default settings"
+elif [ $# -eq 1 ]; then
+  NAMESPACE=$1
+else # number of arguments is equal or greater than 2
+  NAMESPACE=$1
+  THESHOLD_PODS_NUMBER=$2
+fi
+
+echo "Check status of noobaa pods:"
+echo "Namespace: ${NAMESPACE}"
+echo "Number of pods to start using the system is ${THESHOLD_PODS_NUMBER}"
+
+i=1
+while [ "$i" -le ${NUMBER_OF_ITERATIONS} ];
+do
```

Comment:
```suggestion
if [ "$i" -le ${NUMBER_OF_ITERATIONS} ]
then
```

Reply:
Why do you suggest changing it from loop to condition?